packages/frontend-v2/hooks/useFetchSearchCourse.ts

```diff
@@ -0,0 +1,61 @@
+import useSWR, { SWRResponse } from "swr";
```

Comment:
I don't think we use this hook anywhere, since we only fetch 1 course in the getRequiredCourseCoreqs function. And over there we use the fetching function directly.

If yall can make a good argument to keep it I'm down, otherwise might be nice to reduce code.